app/views/providers/index.html.erb

```diff
@@ -37,7 +37,8 @@
         <td><%= provider.provider_id_prefix %></td>
         <td><%= provider.in_production %></td>
         <td class="oai-actions btn-provider"><%= link_to 'Harvest from OAI Source', harvest_provider_path(provider), method: :post, class: :triggerWaiting, data: { confirm: 'This may take a while for OAI seeds with many items.  Are you sure?' } %></td>
-        <td class="oai-actions btn-provider"><%= link_to 'Harvest all OAI Seeds from this Institution', harvest_all_by_institution_provider_path(provider), method: :post, class: :triggerWaiting, data: { confirm: 'This may take a while for institutions with many OAI seeds. Are you sure?' } %></td>
+        <td class="oai-actions btn-provider"><%= link_to 'Harvest all OAI Seeds from this Institution', harvest_all_selective_contributing_institution_provider_path(provider), method: :post, class: :triggerWaiting, data: { confirm: 'This may take a while for institutions with many OAI seeds. Are you sure?' } %></td>
+        <td class="oai-actions btn-provider"><%= link_to 'Harvest all OAI Seeds from this Institution\'s Intermediate Provider', harvest_all_selective_intermediate_provider_provider_path(provider), method: :post, class: :triggerWaiting, data: { confirm: 'This may take a while for intermediate providers with institutions and associated OAI seeds. Are you sure?' } %></td>
```

Comment:
Sounds good!  I added a method for this.  Could be repurposed in this view wholesale if we add more attribute-dependent actions later on?